src/snackbar.tsx

```diff
@@ -40,7 +40,7 @@ export type ImperativeHandle = {
     close: SnackbarCloseHandler;
 };
 
-const SnackbarComponent = React.forwardRef<ImperativeHandle, Props>(
+const SnackbarComponent = React.forwardRef<ImperativeHandle, Omit<Props, 'duration'> & {duration: number}>(
```

Comment:
why? The snackbar component should have the same duration type as the bridge implementation

Reply:
This is an internal component, and not used anywhere else. But I can change it just to keep the code cleaner

Reply:
updated